### PR TITLE
[21754] Homepage Refresh Code Organization

### PR DESCRIPTION
**Pull request by LanceGundersen**
## Description
The homepage refresh code is mixed together with our existing list page code and it could be good to separate it out more:

## Testing done
Local and Unit

## Screenshots
n/a

## Acceptance criteria
- [ ] Separate refresh list page components into separate folder from current homepage list components
- [ ] Make sure all homepage refresh code follows our typical conventions for file and folder structure
- [ ] Make sure homepage refresh tests match component names and folder structure of component code

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by jbalboni** [comment]
@LanceGundersen Can you take a look through the unit test files that are associated with different List components? There are files like FutureAppointmentsListV2 that don't seem to match a real component name (might just be that one).

**Review by vbahinwillit — APPROVED** [review]
LGTM

**Review by vbahinwillit — APPROVED** [review]

**Comment by LanceGundersen** [comment]
@jbalboni I believe I have everything now.

File: src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/ConfirmedAppointmentDetailsPage.unit.spec.jsx [hunk 1]
```diff
@@ -7,21 +7,21 @@ import {
   getVAAppointmentMock,
   getVAFacilityMock,
   getCancelReasonMock,
-} from '../../mocks/v0';
+} from '../../../mocks/v0';
```
Comment (jbalboni) [inline src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/ConfirmedAppointmentDetailsPage.unit.spec.jsx:10]: Following the convention of the rest of the files, both this and the other file in `src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage` should probably be `index` files.

**Comment by jbalboni** [comment]
@LanceGundersen Something else I noticed is that in `src/applications/vaos/appointment-list/components/AppointmentsPage`, `AppointmentListItem`, `ExpressCareListItem`, and `RequestListItem` are all for the refresh, rather than the old homepage, so they shouldn't be in that folder, I don't think.

**Comment by LanceGundersen** [comment]
> @LanceGundersen Something else I noticed is that in `src/applications/vaos/appointment-list/components/AppointmentsPage`, `AppointmentListItem`, `ExpressCareListItem`, and `RequestListItem` are all for the refresh, rather than the old homepage, so they shouldn't be in that folder, I don't think.

Fixed, there seems to be a bunch of these in here

**Review by jbalboni — APPROVED** [review]
Looks like there are some conflicts to clean up, but looks good otherwise.

**Pull request merged**